Packs/BitwardenPasswordManager/Integrations/BitwardenPasswordManager/BitwardenPasswordManager_test.py

```diff
@@ -61,6 +61,69 @@ def test_login_with_valid_token(mock_client_with_valid_token):
     assert mock_client_with_valid_token.token == "access_token"
 
 
+@freeze_time("2024-04-25 00:00:00")
+@pytest.mark.parametrize(
+    "base_url, full_url",
+    [
+        (MOCK_BASEURL, "https://identity.bitwarden.com/connect/token"),
+        ("https://mock.api.eu", "https://identity.bitwarden.eu/connect/token"),
```

Comment:
Add a case like we talked about, were the `.eu` is not at the end and the URL is kept with `.com`